### PR TITLE
Update the design of the `upgrade` button

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
Update the design of the `upgrade` button.

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by pavinduLakshan** [comment]
Shall we add a screenshot of the changes?

**Review by pavinduLakshan — APPROVED** [review]

File: features/admin.core.v1/components/header.tsx [hunk 1]
```diff
@@ -41,24 +41,14 @@ import { IdentifiableComponentInterface, ProfileInfoInterface } from "@wso2is/co
...
-import React, {
-    FunctionComponent,
-    ReactElement,
-    ReactNode,
-    useContext,
-    useEffect,
-    useMemo,
-    useState
-} from "react";
+import React, { FunctionComponent, ReactElement, ReactNode, useContext, useEffect, useMemo, useState } from "react";
```
[the remark this answers is not in the document]
Comment (Achintha444) [reply features/admin.core.v1/components/header.tsx:44]: this was done from formatting the document. Since no imports is missing this is okay right ?

**Review by pavinduLakshan — APPROVED** [review]

File: features/admin.core.v1/assets/icons/diamond.svg [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!--
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (pavinduLakshan) [inline features/admin.core.v1/assets/icons/diamond.svg:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.core.v1/assets/icons/diamond.svg [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<!-- TODO: Need to remove this -->
```
[the remark this answers is not in the document]
Comment (Achintha444) [reply features/admin.core.v1/assets/icons/diamond.svg:19]: this is needed, as this file will be removed with bumping the new oxygen version

File: features/admin.core.v1/assets/icons/diamond.svg (hunk 2 above)
Comment (Achintha444) [reply features/admin.core.v1/assets/icons/diamond.svg]: Fixed with https://github.com/wso2/identity-apps/pull/6559/commits/0cc5b386ec55c2ecf76c19c40652387dfe6e71cd

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**